spec/system/curator_controlled_metadata_spec.rb

```diff
@@ -24,7 +24,7 @@
       expect(page).to have_field("ark", readonly: true)
       expect(page).to have_field("resource_type", readonly: true)
       expect(page).to have_field("resource_type_general", disabled: true)
-      expect(page).to have_field("version_number", disabled: true)
+      expect(page).to have_field("version_number", disabled: false)
```

Comment:
@leefaisonr I don't think the Princeton Submitter should be able to change the version number.

Reply:
Thanks @carolyncole - I have made those changes.